api/pom.xml

```diff
@@ -237,13 +247,17 @@
     </profiles>
 
     <build>
-        <finalName>${project.artifactId}</finalName>
+        <!--finalName>${project.artifactId}</finalName-->
```

Comment:
remove

Reply:
Remved. All projects use default names with versions. I see no reason why this one should have an exception.